src/map/ai/controllers/automaton_controller.cpp

```diff
@@ -1667,4 +1669,26 @@ namespace autoSpell
             return {};
         }
     }
-} // namespace autoSpell
+
+    void LoadAutomatonAbilities()
+    {
+        const char* Query = "SELECT abilityid, abilityname, reqframe, skilllevel FROM automaton_abilities;";
+
+        int32 ret = Sql_Query(SqlHandle, Query);
+
+        if (ret != SQL_ERROR && Sql_NumRows(SqlHandle) != 0)
+        {
+            while (Sql_NextRow(SqlHandle) == SQL_SUCCESS)
+            {
+                uint16           id = (uint16)Sql_GetUIntData(SqlHandle, 0);
+                AutomatonAbility PAbility{ (uint8)Sql_GetUIntData(SqlHandle, 2), (uint16)Sql_GetUIntData(SqlHandle, 3) };
+
+                autoAbilityList[id] = std::move(PAbility);
+
+                auto filename = fmt::format("./scripts/globals/abilities/pets/automaton/{}.lua", Sql_GetData(SqlHandle, 1));
+                ShowDebug("Caching Automaton Ability: %s\n", filename);
```

Comment:
Can we get rid of this additional logging?

Reply:
Whoops! Absolutely